.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 80,
```

Comment:
I think printWidth 80 is too small. Most people use large screens and laptops >= 14 inch. Can we make this 100 or 120?

Reply:
prettier's printWidth is not a maximum line length limit. See their explanation on the subject: https://prettier.io/docs/en/options.html#print-width